thorlcr/graph/thgraphmaster.cpp

```diff
@@ -675,6 +675,7 @@ cost_type CMasterActivity::calcFileReadCostStats(bool updateFileProps)
         return readCost;
```

Comment:
if useJhtreeCacheStats is true, should numReads be set to numActualReads?  I suspect so (affects the return value and the call to updateCostAndNumReads()).

If so the code could be simplified and could avoid the call to getClusterName() if numReads==0